tools/src/tools.ml

```diff
@@ -257,7 +280,40 @@ let typeDetail typ ~env ~full =
 let makeId modulePath ~identifier =
   identifier :: modulePath |> List.rev |> SharedTypes.ident
 
-let extractDocs ~path ~debug =
+(**
+Get relative path to res/resi file
+*)
+let findRelativePath ~rootPath ~path =
+  let open Filename in
+  let realPath =
+    match rootPath = "." with
+    | true -> Sys.getcwd ()
+    | false ->
+      if is_relative rootPath then concat (Sys.getcwd ()) rootPath else rootPath
+  in
+  let rec loop dirPath acc =
+    match dirname dirPath = realPath with
+    | true -> basename dirPath :: acc |> String.concat "/"
+    | false -> (
+      match dirPath with
+      | "/" -> failwith "Failed to find relative path of package"
+      | "." -> path
+      | _ -> loop (dirname dirPath) (basename dirPath :: acc))
+  in
+
+  loop path []
```

Comment:
No built in fn that does this I take it...?

Reply:
thank you. Found in `Analysis.Files.realpath`

5396f79b

Reply:
Great! Now we just need to figure out the tests for Windows. Paths being different is obviously expected.